db

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+###
+# Command line tool for managing Open Vault database
+###
+import click
+from utils import run, get_pod_name
+from datetime import datetime
+
+
+@click.group()
+@click.option(
+    '--context',
+    '-c',
+    required=True,
+    help='the kubectl context to use',
+    type=click.Choice(['openvault', 'openvault-demo']),
+)
+def cli(context: str):
+    """Database CLI
+
+    Management of Open Vault production databases
+    """
+    run(f'kubectl config use-context { context }')
+    click.echo(f'context set to { context }')
+
+
+@cli.command()
+@click.pass_context
+def backup(ctx):
```

Comment:
where is `ctx` coming from here?

Reply:
@afred It's coming from the `@click.pass_context` decorator, from click: https://click.palletsprojects.com/en/8.1.x/commands/#nested-handling-and-contexts

Reply:
👍